terraform/rancher/cluster/helm.tf

```diff
@@ -28,27 +36,27 @@ resource "helm_release" "alb_controller" {
   }
 }
 
-#CSI interface used by Container Orchestrators to manage the lifecycle of Amazon EBS volumes.
-resource "helm_release" "aws_ebs_csi" {
-  depends_on = [module.eks_cluster, module.ebs_csi_irsa_role]
-  name       = "aws-ebs-csi-driver"
-  namespace  = "kube-system"
-  repository = "https://kubernetes-sigs.github.io/aws-ebs-csi-driver"
-  chart      = "aws-ebs-csi-driver"
-  version    = "2.12.1"
-  set {
-    name  = "controller.region"
-    value = var.aws_region
-  }
-  set {
-    name  = "controller.serviceAccount.name"
-    value = "aws-ebs-csi-driver"
-  }
-  set {
-    name  = "controller.serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
-    value = module.ebs_csi_irsa_role.iam_role_arn
-  }
-}
+##CSI interface used by Container Orchestrators to manage the lifecycle of Amazon EBS volumes.
+#resource "helm_release" "aws_ebs_csi" {
```

Comment:
Are we going to fully abandon stateful applications in the cluster?

Reply:
Nah, this application was removed here because AWS added it to the official addons  list
https://docs.aws.amazon.com/eks/latest/userguide/eks-add-ons.html